### ISSUE TITLE
Missing in lokalise "discharging", "none"

### ISSUE THREAD

**Issue by misa1515** [home-assistant/frontend#17366]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

Missing two words in lokalise for translation
![1689858543227_discharging](https://github.com/home-assistant/frontend/assets/61636045/b82043f0-ef05-4442-b33f-8be4db4e05aa)

### Describe the behavior you expected

Add missing words to lokalise

### Steps to reproduce the issue

1.
2.
3.
...

### What version of Home Assistant Core has the issue?

2023.07.1

**Comment by frenck** [comment]
I'm not sure what I am looking at @misa1515. Please provide me with the steps I need to take to get to this screen (in plain English).

../Frenck

**Comment by misa1515** [comment]
@frenck you can found it in this submenu
Settings/Companion application (location and notifications)/Manage Sensors/

**Comment by karwosts** [comment]
May be at least partially duplicate with #17205

**Comment by MickLesk** [comment]
Yes, its a duplicate of my issue. Unfortunately, I have not yet been able to find a proper solution.

**Comment by frenck** [comment]
Closing as a duplicate.
